eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1919,7 +1919,12 @@ public function removeTranslation(ContentInfo $contentInfo, $languageCode)
             if (count($versionInfo->languageCodes) < 2) {
                 throw new BadStateException(
                     '$languageCode',
-                    'Specified translation is the only one Content Object Version has'
+                    sprintf(
+                        'The Version (ContentId=%d, VersionNo=%d) has the only one, %s, Translation. Remove this Version before proceeding.',
```

Comment:
note, there might be several versions with only one language, so maybe adjust error to account for that so person doesn't not expect it is all good after deleting the once version.

Reply:
by adjusting this msg for single Version saying that there might be more or by collecting single language versions and concatenating them into one error msg?

Reply:
both can work, latter would provide more info to user and hence be more user friendly if we can make the string understandable

Reply:
Going with the latter one, how about: c1b00fc